gravitee-am-ui/src/app/domain/settings/theme/theme.component.ts

```diff
@@ -13,13 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import {Component, OnInit} from '@angular/core';
+import {Component, ElementRef, OnInit, ViewChild} from '@angular/core';
 import {ActivatedRoute, Router} from '@angular/router';
 import {SnackbarService} from '../../../services/snackbar.service';
 import {AuthService} from '../../../services/auth.service';
 import {ThemeService} from '../../../services/theme.service';
 import {DialogService} from '../../../services/dialog.service';
 import * as _ from 'lodash';
+import {FormService} from "../../../services/form.service";
```

Comment:
should be single quote

Reply:
FP

Reply:
@farmborough, the issue is with TSLint configuration. It should be updated. Being said that this link is bit informative :https://stackoverflow.com/questions/47490491/import-statements-single-vs-double-quotes [no idea why google prefers single quote though]